exercicios/para-casa/entregas/amanda-santos/AuthorExample.js

```diff
@@ -0,0 +1,35 @@
+class AuthorExample {
+   
```

Comment:
Olá Amanda 

Senti falta da declaração dos atributos privados como sugere no enunciado, métodos privados deve ser declarados antes do constructor com o uso de uma # antes do nome do atributo, como no exemplo 
`class Author {` 
` #nome` 
`constructor(nome) {` 
`this.#nome = nome `
`}` 
`}`